lib/verify.js

```diff
@@ -55,10 +55,10 @@ module.exports = async (pluginConfig, context) => {
     const github = getClient({githubToken, githubUrl, githubApiPathPrefix, proxy});
 
     // https://github.com/semantic-release/github/issues/182
-    // Do not check for permissions in GitHub actions, as the provided token is an installation access token.
-    // github.repos.get({repo, owner}) does not return the "permissions" key in that case. But GitHub Actions
+    // Do not check for permissions in GitHub actions nor in Github Apps, as the provided token is an installation access token.
+    // github.repos.get({repo, owner}) does not return the "permissions" key in that case. But GitHub Actions and Apps
     // have all permissions required for @semantic-release/github to work
-    if (env.GITHUB_ACTION) {
+    if (env.GITHUB_ACTION || env.GITHUB_APP) {
```

Comment:
where does the `GITHUB_APP` environment variable come from? If it's a custom environment variable that you set in your particular case, then I don't think we should add a check for it to semantic-release